easybuild/easyblocks/o/orca.py

```diff
@@ -86,14 +86,21 @@ def install_step(self):
                 (['auto*', 'orca*', 'otool*'], 'bin'),
                 (['*.pdf'], 'share'),
             ]
-            # Version 5 extra files
-            if LooseVersion(self.version) >= LooseVersion('5.0.0'):
-                compoundmethods = (['ORCACompoundMethods'], 'bin')
-                files_to_copy.append(compoundmethods)
-            # Shared builds have additional libraries
-            libs_to_copy = (['liborca*'], 'lib')
-            if all([glob.glob(p) for p in libs_to_copy[0]]):
-                files_to_copy.append(libs_to_copy)
+
+            # Version 6 extra files
+            if LooseVersion(self.version) >= LooseVersion('6.0.0'):
+                files_to_copy.extend(['datasets', 'lib', (['CompoundScripts'], 'bin')])
+
+            else:
```

Comment:
Should this really be an "else"? Are the extra files from version 5 no longer available in version 6?

Reply:
yes it should.
`ORCACompoundMethods` no longer exists, it is now `CompoundScripts`, and `liborca*` is now in a `lib` dir, so we can just copy over `lib`.